### DISCUSSION TITLE
About the public network access of the self-managed Rustdesk Remote Desktop server: The relay server cannot be connected

### DISCUSSION THREAD

**Discussion by GaleNuo** [rustdesk/rustdesk#10379] [General]
My self-built server is deployed on the private network, and the private IP address is set in for the private client and the public IP address is set in for the external client. However, in this setting, the extranet client cannot connect to the relay server, but the intranet client is used normally. However, I have to have an intranet client that can't access the Internet, so I can't fill in the extranet IP. Pls how can I fix this.

**Comment by rustdesk** [comment]
#763
